ui/feed_mark_as_read.go

```diff
@@ -33,5 +33,10 @@ func (h *handler) markFeedAsRead(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	html.Redirect(w, r, route.Path(h.router, "feeds"))
+	redirectToSelf, ok := r.URL.Query()["redirect_to_self"]
```

Comment:
Use `request.QueryStringParam()` from https://github.com/miniflux/v2/blob/master/http/request/params.go

Or use `r.URL.Query().Get("redirect_to_self")`

Reply:
Good point, I wasn't aware of that helper function, thanks!